test/functional/wicked_pdf_helper_assets_test.rb

```diff
@@ -6,18 +6,23 @@ class WickedPdfHelperAssetsTest < ActionView::TestCase
   include WickedPdfHelper::Assets
 
   if Rails::VERSION::MAJOR == 4
-    test 'wicked_pdf_asset_path should return an url when assets are served by an asset server' do
+    test 'wicked_pdf_asset_path should return a url when assets are served by an asset server' do
       expects(:asset_pathname => 'http://assets.domain.com/dummy.png')
       assert_equal 'http://assets.domain.com/dummy.png', wicked_pdf_asset_path('dummy.png')
     end
 
-    test 'wicked_pdf_asset_path should return an url with a protocol when assets are served by an asset server with relative urls' do
+    test 'wicked_pdf_asset_path should return a url when assets are served by an asset server using HTTPS' do
```

Comment:
Quick test to demo out this error -- if we change it back to source[0,7] in the helper then this test will fail, changing it to source[0,8] in the helper will make it pass again :confetti_ball: